internal/app/app.go

```diff
@@ -167,8 +168,15 @@ func (c *CacheApp) RunApp(stopCh <-chan struct{}) {
 	// https://github.com/kubernetes/dns/issues/282
 	// sometimes the interface gets the ip and then loses it, if added too soon.
 	c.runChecks()
-	// run periodic blocks
-	c.runPeriodic()
+
+	if c.params.Daemon {
+		// run periodic blocks
+		c.runPeriodic()
+	} else {
+		klog.Infoln("Not running as a daemon")
```

Comment:
can we skip the else here and just log "running as a daemon" when daemon mode is enabled?